addons/mail/static/src/components/thread_needaction_preview/thread_needaction_preview.xml

```diff
@@ -40,11 +40,11 @@
                         </t>
                     </div>
                     <div class="o_ThreadNeedactionPreview_core d-flex align-items-baseline">
-                        <span class="o_ThreadNeedactionPreview_coreItem o_ThreadNeedactionPreview_inlineText text-truncate" t-att-class="{ 'o-empty': inlineLastNeedactionMessageAsOriginThreadBody.length === 0 }">
+                        <span class="o_ThreadNeedactionPreview_coreItem o_ThreadNeedactionPreview_inlineText text-truncate" t-att-class="{ 'o-empty': threadNeedactionPreviewView.isEmpty }">
                             <t t-if="threadNeedactionPreviewView.messageAuthorPrefixView">
                                 <MessageAuthorPrefix localId="threadNeedactionPreviewView.messageAuthorPrefixView.localId"/>
                             </t>
-                            <t t-esc="inlineLastNeedactionMessageAsOriginThreadBody"/>
+                            <t class="o_ThreadNeedactionPreview_coreItem o_ThreadNeedactionPreview_messageBody" t-esc="inlineLastNeedactionMessageAsOriginThreadBody"/>
```

Comment:
On what are these classes applied? It sounds a bit unclear to me, I wonder if it's the right place to define them. AFAIK `<t/>` tags do not map to any actual element in the DOM

Reply:
It is used for replacing the rating face URL when doing the inherited stuff in rating.